rsc/settings.properties.dist

```diff
@@ -1,50 +1,70 @@
+###########################################
 # This file contains Open Hospital settings
+###########################################
+# external settings
 MODE=OH_MODE
+###########################################
+# internal settings, modules, directories
 LANGUAGE=OH_LANGUAGE
 SINGLEUSER=YES_OR_NO
+DEBUG=no
+DOC_DIR=OH_DOC_DIR
+PATIENTPHOTOSTORAGE=PHOTO_DIR
+INTERNALVIEWER=yes
+SMSENABLED=no
+VIDEOMODULEENABLED=yes
+XMPPMODULEENABLED=no
+###########################################
+# application settings
+ENHANCEDSEARCH=no
+INTERNALPHARMACIES=yes
+LABEXTENDED=yes
+LABMULTIPLEINSERT=yes
+MATERNITYRESTARTINJUNE=no
+MERGEFUNCTION=yes
+OPDEXTENDED=yes
+PATIENTEXTENDED=yes
+PATIENTVACCINEEXTENDED=yes
+###########################################
+# GUI settings
+MAINMENUALWAYSONTOP=no
+###########################################
+# accounting
+ALLOWMULTIPLEOPENEDBILL=yes
+ALLOWPRINTOPENEDBILL=yes
+BILLSREPORT=BillsReport
+BILLSREPORTMONTHLY=BillsReportMonthly
+BILLSREPORTPENDING=BillsReportPending
+PATIENTBILL=PatientBill
+PATIENTBILLGROUPED=PatientBillGrouped
+PATIENTBILLSTATEMENT=PatientBillStatement
+RECEIPTPRINTER=yes
+###########################################
+# pharmacy
 AUTOMATICLOT_IN=no
 AUTOMATICLOT_OUT=no
 AUTOMATICLOTWARD_TOWARD=no
 LOTWITHCOST=yes
-VISITSHEET=WardVisits
-PATIENTSHEET=patient_clinical_sheet_ver3
-EXAMINATIONCHART=patient_examination
-OPDCHART=patient_opd_chart
-ADMCHART=patient_adm_chart
-DISCHART=patient_dis_chart
-PATIENTBILL=PatientBill
-BILLSREPORT=BillsReport
-BILLSREPORTPENDING=BillsReportPending
-BILLSREPORTMONTHLY=BillsReportMonthly
 PHARMACEUTICALORDER=PharmaceuticalOrder
 PHARMACEUTICALSTOCK=PharmaceuticalStock_ver4
 PHARMACEUTICALSTOCKLOT=PharmaceuticalStock_ver5
 PHARMACEUTICALAMC=PharmaceuticalAMC
-PATIENTEXTENDED=yes
-OPDEXTENDED=yes
-MATERNITYRESTARTINJUNE=no
-LABEXTENDED=yes
-LABMULTIPLEINSERT=yes
-INTERNALPHARMACIES=yes
-MERGEFUNCTION=yes
-SMSENABLED=no
-INTERNALVIEWER=yes
-DOC_DIR=OH_DOC_DIR
-MAINMENUALWAYSONTOP=no
-RECEIPTPRINTER=yes
-VIDEOMODULEENABLED=yes
-PATIENTVACCINEEXTENDED=yes
-ENHANCEDSEARCH=no
-XMPPMODULEENABLED=no
+###########################################
+# dicom / imaging settings
```

Comment:
FWIW,  dicom comments in other files do not include "imaging"; should they?

Reply:
Yes, I think so, we should correct them when we find occurrences,

Reply:
this file is for users (other files are scripts), it could just say "imaging" since we changed in this way in the doc (and not the code) but since it contains DICOM* parameters, I would leave it as "crossover" temporary solution.